doc/DataPlaneCodeGeneration/Azure_SDK_Package_Ship_Requirements.md

```diff
@@ -0,0 +1,65 @@
+# Azure sdk package ship requirements
+
+Before the library package can be released, you will need to add several requirements manually, including tests, samples, README, and CHANGELOG.
+You can refer to following guideline to add those requirements:
+
+- Tests: <https://azure.github.io/azure-sdk/general_implementation.html#testing>
+- README/Samples: <https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-repository>
+- Changelog: <https://azure.github.io/azure-sdk/policies_releases.html#change-logs>
+- Other release concerns: <https://azure.github.io/azure-sdk/policies_releases.html>
+
+## Tests
+
+In this section, we will talk about adding unit tests and live tests and how to run them. You will notice that there is a test project under `Azure.<group>.<service>\tests`.
+
+Here is the step by step process to add tests:requirements
+
+- Add other client parameters in `<client-name>ClientTestEnvironment.cs`
+- Update `<client-name>ClientTest.cs`.
+  - Please refer to [Using the TestFramework](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md) to add tests.
+
```

Comment:
Do we do any validation on the test health? coverage numbers etc?

Reply:
No. We will run this test, but no coverage. We will plane to support this later.